pkg/actor/validate_version.go

```diff
@@ -312,6 +327,32 @@ func IsJobPodRunning(
 	return nil
 }
 
+func IsContainerStatusImagePullBackoff(
+	ctx context.Context,
+	clientset kubernetes.Interface,
+	job *kbatch.Job,
+	l logr.Logger,
+	image string,
+) error {
+	//get pod for the job we created
+	pods, err := clientset.CoreV1().Pods(job.Namespace).List(ctx, metav1.ListOptions{
+		LabelSelector: labels.Set(job.Spec.Selector.MatchLabels).AsSelector().String(),
```

Comment:
Can this be “”? Should we pass in the entire job since we just need a label selector? We could move this into a helper fun maybe.

Reply:
@ yes we can do something more generic... we can add an issue for this